gum/backend-x86/gumstalker-x86.c

```diff
@@ -3064,6 +3064,12 @@ gum_exec_block_virtualize_branch_insn (GumExecBlock * block,
     gpointer is_false;
     GumExecCtxReplaceCurrentBlockFunc regular_entry_func, cond_entry_func;
 
+    if (insn->ci->id == X86_INS_JMP)
+    {
+      if (ctx->stalker->any_probes_attached)
+        gum_exec_block_write_call_probe_code (block, &target, gc);
+    }
```

Comment:
We need to move this after the `if (is_conditional)` block below, so that we don't do the probing for conditional branches that are not taken.